src/IssueViz/Editor/SelectedIssueTagging/Adornment/IssueLocationAdornment.cs

```diff
@@ -58,6 +58,11 @@ public IssueLocationAdornment(IAnalysisIssueLocationVisualization locationViz, I
 
         public void Update(IFormattedLineSource formattedLineSource)
         {
+            if (formattedLineSource == null)
```

Comment:
How did we miss this before? Is it because it's timing related?

If so, it would be worth adding a comment. Also, it's probably worth calling out that the impact is just cosmetic - the colours/font might not be updated correctly.

Reply:
Yep, it's timing related. It seems it can happen if the tagger is somehow called before the View finished initializing. Ugras was unable to reproduce it on his machine and it's the first time it happened on mine.